src/operators.jl

```diff
@@ -1,8 +1,8 @@
 ## Lifted operators
 
-importall Base.Operators
-import Base: promote_op, abs, abs2, sqrt, cbrt, scalarmin, scalarmax, isless
-using Compat: @compat
+import Base: !, %, &, *, +, -, /, //, <, <<, <<<, ==, ===, >, >>, >>>, \, ^, getindex, |, ~,
```

Comment:
Did a `setdiff` and it looks like this covers all the operators being overloaded and the only extras are `Set(Symbol[:\, ://, :(===)])` which aren't unreasonable.